### PR TITLE
Clown makes a sad trombone noise on death

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
rscadd: The clown will play a sad trombone noise upon death.
/:cl:

### CODE REVIEW

**Comment by bgobandit** [comment]
:postal_horn:

**Comment by lzimann** [comment]
next step is making all clowns valid

File: code/game/objects/items/weapons/implants/implant_clown.dm [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+/obj/item/weapon/implant/sad_trombone/trigger(emote, mob/source)
+	if(emote == "deathgasp")
+		playsound(loc, 'sound/misc/sadtrombone.ogg', 50, 0)
+
+/obj/item/weapon/implanter/sad_trombone
+	name = "implanter (sad_trombone)"
+
+/obj/item/weapon/implanter/sad_trombone/New()
+	imp = new /obj/item/weapon/implant/explosive(src)
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/implants/implant_clown.dm]: nice

Comment (PKPenguin321) [inline code/game/objects/items/weapons/implants/implant_clown.dm]: bobo fall down go boom

**Comment by Bawhoppen** [comment]
niceone kor

File: code/game/objects/items/weapons/implants/implant_clown.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/game/objects/items/weapons/implants/implant_clown.dm]: uh oh

**Comment by GunHog** [comment]
Make this build-able in Science with bananium. PLEASE!!!!!!!!!!!!!!!!!!!!!!

**Comment by bgobandit** [comment]
leo why do you want to be valid

**Comment by lordpidey** [comment]
Wait, why the fun removal tag?  This is literally the opposite of the fun removal tag.

**Comment by LatD** [comment]
Fun addition

**Comment by bgobandit** [comment]
the clown, who represents fun, dies, i.e. is removed

**Comment by ike709** [comment]
"Reverse Fun Removal" tag

**Comment by KorPhaeron** [comment]
@AnturK 

Here is my merge token #18257

It has a dumb name because Cheridan is a bully but still

**Comment by RemieRichards** [comment]
Why didn't you use ```implant()```?

**Pull request merged**